jenkins-ansible/blueprint.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: xl/v1
+kind: Blueprint
+metadata:
+  projectName: Extended Jenkins Ansible Pipeline
+  description: |
+    This is a simple release tempalte that uses Ansible and Jenkins.
+  author: XebiaLabs
+  version: 1.0
+parameters:
+  - name: folderName
```

Comment:
cant we fix folder name as its a sample app

Reply:
yes, made a fixed folder name